.github/workflows/main.yml

```diff
@@ -39,15 +41,17 @@ jobs:
         run: |
           sudo locale-gen en_US.UTF-8 nl_NL@euro
           python -m venv .
+          ln -s requirements-${{ matrix.config[2] }}.txt requirements.txt
           bin/pip install -r requirements.txt
+          ln -s versions-${{ matrix.config[2] }}.cfg versions.cfg
           bin/buildout -c .github.cfg
       - name: Pre commit
         uses: pre-commit/action@v2.0.0
-        if: ${{ matrix.config[1] == '3.8' }}
+        if: ${{ matrix.config[1] == '3.11' }}
```

Comment:
3.11 is not in the matrix.

Reply:
The 3.11 is planned but commented some lines above